plugins/chrome-ux-report/config.d.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * ChromeUXReport schema below is an abstract of what's used within chrome-ux-report-backend
+ */
+export interface Config {
+  /**
+   * @visibility frontend
+   */
+  chromeUXReport: {
+    /**
+     * @visibility frontend
+     */
+    projectId: string;
+    /**
+     * @visibility frontend
+     */
+    keyPath: string;
```

Comment:
Only list the config that the page actually uses, I'm assuming the frontend plugin does not need the key path :grin: